packages/sui-studio/src/components/demo/index.js

```diff
@@ -139,34 +174,73 @@ export default class Demo extends Component {
           <ContextButtons
             ctxt={ctxt || {}}
             selected={ctxtSelectedIndex}
-            onContextChange={this.handleContextChange} />
+            onContextChange={this.handleContextChange}
+          />
           <ThemesButtons
             themes={themes}
             selected={themeSelectedIndex}
-            onThemeChange={this.handleThemeChange} />
+            onThemeChange={this.handleThemeChange}
+          />
           <EventsButtons events={events || {}} store={store} domain={domain} />
         </div>
 
         <button className='sui-StudioDemo-codeButton' onClick={this.handleCode}>
-          {isCodeOpen ? iconClose : iconCode }
+          {isCodeOpen ? iconClose : iconCode}
+        </button>
+
+        <button
+          className='sui-StudioDemo-fullScreenButton'
+          onClick={this.handleFullScreen}
+        >
+          {isFullScreen ? iconFullScreenExit : iconFullScreen}
         </button>
 
-        {isCodeOpen && <CodeEditor
-          isOpen={isCodeOpen}
-          onChange={playground => { this.setState({playground}) }}
-          playground={playground}
-        />}
+        {isCodeOpen && (
+          <CodeEditor
+            isOpen={isCodeOpen}
+            onChange={playground => {
+              this.setState({ playground })
+            }}
+            playground={playground}
+          />
+        )}
 
         <Preview
           code={playground}
-          scope={{React, [`${cleanDisplayName(Enhance.displayName)}`]: Enhance, domain, ...nonDefaultExports}}
+          scope={{
+            React,
+            [`${cleanDisplayName(Enhance.displayName)}`]: Enhance,
+            domain,
+            ...nonDefaultExports
+          }}
         />
       </div>
     )
   }
 
   handleCode = () => {
-    this.setState({isCodeOpen: !this.state.isCodeOpen})
+    this.setState({ isCodeOpen: !this.state.isCodeOpen })
+  }
+
+  handleFullScreen = () => {
+    this.setState({ isFullScreen: !this.state.isFullScreen }, () => {
+      const { isFullScreen } = this.state
+      const aside = document.querySelector('.sui-Studio-sidebar')
+      const toolBar = document.querySelector('.sui-StudioWorkbench-navigation')
+      const secundaryToolBar = document.querySelector(
+        '.sui-StudioNavBar-secondary'
+      )
+
+      if (isFullScreen) {
+        aside.classList.add('hidden')
```

Comment:
Always better to add a single class once that will have several instructions.
For instance add ".sui-Studio--fullscreen" to the main element and hide the subelements by css.

Reply:
That brake the componetization way of think. Isn’t ?!  Because I display a component depending of my context in the page.

Reply:
You won't if you make component selectors depending on the global "classname", so that in your CSS will have something like this:
```css
.sui-Studio--fullscreen {
  & .aside-classname { display: none; }
}
```
And the same for the other components. Isn't it @davidbarna?

Reply:
@carlosvillu to me what breaks the "componetization way of think" its to access remote components and alter their DOM model adding classnames.
I'm just saying to do it by css like @kikoruiz illustrated. It's less intrusive.

My main concern is performance. AFAIK, adding classes several times provokes several renders of the browser. It may be optimised as it's in the same call stack. But I it was I would still applied the mention solution. It'd still be less code and a much clearer approach to me.